app/assets/javascripts/models/page/search_page.js.coffee

```diff
@@ -12,6 +12,8 @@
 #= require models/ui/state_manager
 #= require models/ui/feedback
 #= require models/ui/sitetour
+#= require models/ui/reverb_retirement
+#= require util/js.cookies
```

Comment:
Require the js.cookies right after jquery loads which means a different file location for js.cookies is desired.

Reply:
I'll need to know where that is :\